Wikipedia/Code/ProfileCoordinator.swift

```diff
@@ -175,9 +175,39 @@ final class ProfileCoordinator: NSObject, Coordinator, ProfileCoordinatorDelegat
     }
     
     private func showYearInReview() {
+        if let viewModel, !viewModel.isUserLoggedIn() {
+            presentLoginPrompt()
+        }
         yirCoordinator.start()
     }
     
+    private func presentLoginPrompt() {
+        let title = WMFLocalizedString("profile-year-in-review-login-title", value: "Log in/Join Wikipedia", comment: "Title of alert that asks user to login if they are entering Year in Review.")
```

Comment:
This value should be "Log in for access to Year in Review"? Based on task requirements.